fission/src/ui/components/MainHUD.tsx

```diff
@@ -23,7 +23,14 @@ const MainHUDButton: React.FC<ButtonProps> = ({ value, icon, onClick, larger })
     return (
         <Button
             onClick={onClick}
-            className={`relative flex flex-row cursor-pointer bg-background w-full m-auto px-2 py-1 text-main-text border-none rounded-md ${larger ? "justify-center" : ""} items-center hover:brightness-105 focus:outline-0 focus-visible:outline-0`}
+            className={`relative flex flex-row 
+                cursor-pointer 
+                bg-background w-full m-auto px-2 py-1 text-main-text border-none rounded-md ${larger ? "justify-center" : ""} 
+                items-center hover:brightness-105 focus:outline-0 focus-visible:outline-0 
+                transform 
+                transition-transform 
+                hover:scale-[1.015] 
+                active:scale-[1.03]`}
```

Comment:
<img width="240" alt="Screenshot 2024-08-13 at 11 30 55 AM" src="https://github.com/user-attachments/assets/a67a7982-fe8c-4644-a43f-ca400d62725f">

im not a huge fan of this button and its positioning